doit/cmd_resetdep.py

```diff
@@ -65,7 +65,13 @@ def _execute(self, pos_args=None):
                 continue
 
             task.values = values
-            self.dep_manager.save_success(task, result_hash=result)
+
+            try:
+                self.dep_manager.save_success(task, result_hash=result)
+            except OSError as e:
+                write("failed {} ({})\n".format(task.name, e))
```

Comment:
This the Error message will be different than the your use-case when you removed only `line_3.txt`. 

I would prefer if we explicitly check all `file_deps` exist. So we wont even need to check for `res.status == "error"`.